test/gui/shared/steps/steps.py

```diff
@@ -327,8 +328,9 @@ def collaboratorShouldBeListed(
 
 @When('the user waits for the files to sync')
 def step(context):
+    snooze(20)
```

Comment:
I think that static wait without reasonable explanation should be removed. In fact, there should be some change in this step definition. @saw-jan did the change AFAIK.

Reply:
Yeah, I did in this PR https://github.com/owncloud/client/pull/9442